coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -30,8 +30,9 @@ def __init__(self, documentation, language,
         :param range:         The position range of type TextRange.
         """
         self.documentation = documentation
-        self.language = language.lower()
-        self.docstyle = docstyle.lower()
+        self.metadata = docstyle_definition.metadata
+        self.language = docstyle_definition.language.lower()
+        self.docstyle = docstyle_definition.docstyle.lower()
```

Comment:
we don't want to duplicate information. As the information resides already inside the docstyle, we just need to refer to it instead of copying the string reference.